### PR TITLE
Replace all instances of `Const#EOL` with `System#lineSeparator()` #8299

### LINKED ISSUE #8299: Replace all instances of `Const#EOL` with `System#lineSeparator()`
It is preferable not to use a constant for a line separator and the standard library offers a much better alternative.

### PR DESCRIPTION
**Pull request by shivanshsoni**
Fixes #8299

### CODE REVIEW

**Comment by tshradheya** [comment]
Your PR description should be something like this `Fixes #8299`

Refer to [this](https://help.github.com/articles/closing-issues-using-keywords) for other keywords that can be used

**Comment by shivanshsoni** [comment]
Anyone, please tell me what errors are coming now. Why are Travis checks failing?

**Comment by craaaa** [comment]
Your code fails the checkstyle test. You can run `./gradlew lint` locally to make sure your code complies with our coding style before pushing and to get more verbose debugging information -- the recommended workflow is [here](https://github.com/TEAMMATES/teammates/blob/master/docs/process.md).

To find out why Travis is failing, you can click `Details` and check the Travis log :)

**Comment by shivanshsoni** [comment]
Done :-)

**Comment by tshradheya** [comment]
Before any of the senior devs review this, you should try removing the `public static final EOL` from Const.java, to ensure that you have replaced all instances of `Const.EOL` with `System#lineSeparator`

**Comment by shivanshsoni** [comment]
@tshradheya Is it good now?

**Comment by tshradheya** [comment]
Yeah. Although you will have to wait for a senior dev to do a proper review. Just to confirm, you have replaced all instances, try doing a project-wide search using `Ctrl + Shift +F` for IntelliJ to see that you cant find any other instance of `Const.EOL`

**Comment by shivanshsoni** [comment]
Thanks, Sir but I have already done that.

File: src/test/java/teammates/test/driver/GmailServiceMaker.java [hunk 1]
```diff
@@ -23,7 +23,7 @@
 import com.google.api.services.gmail.Gmail;
 import com.google.api.services.gmail.GmailScopes;
 
-import teammates.common.util.Const;
+//import teammates.common.util.Const;
```
Comment (tshradheya) [inline src/test/java/teammates/test/driver/GmailServiceMaker.java]: Remove the import statement rather than commenting it out.

Comment (shivanshsoni) [reply src/test/java/teammates/test/driver/GmailServiceMaker.java]: Ok Sir

File: src/test/java/teammates/test/cases/logic/StudentsLogicTest.java [hunk 2]
```diff
@@ -119,9 +119,17 @@ private void testEnrollStudent() throws Exception {
 
         //create fresh test data
         accountsLogic.createAccount(
-                new AccountAttributes(instructorId, "ICET Instr Name", true,
-                        "instructor@icet.tmt", "TEAMMATES Test Institute 1",
-                        StudentProfileAttributes.builder().withGoogleId(instructorId).withShortName("ICET").build()));
+                AccountAttributes.builder()
```
Comment (tshradheya) [inline src/test/java/teammates/test/cases/logic/StudentsLogicTest.java]: I can find some changes which are unrelated to the PR. Update your branch with the latest master branch

File: src/main/java/teammates/logic/core/CoursesLogic.java [hunk 3]
```diff
@@ -103,8 +103,9 @@ public void createCourseAndInstructor(String instructorGoogleId, String courseId
         } catch (EntityAlreadyExistsException | InvalidParametersException e) {
             //roll back the transaction
             coursesDb.deleteCourse(courseId);
-            String errorMessage = "Unexpected exception while trying to create instructor for a new course " + Const.EOL
-                                  + instructor.toString() + Const.EOL
+            String errorMessage = "Unexpected exception while trying to create instructor for a new course "
+                                  + System.lineSeparator()
+                                  + instructor.toString() + System.lineSeparator()
                                   + TeammatesException.toStringWithStackTrace(e);
```
Comment (tshradheya) [inline src/main/java/teammates/logic/core/CoursesLogic.java]: Line 107 and 108 can be combined?

File: src/main/java/teammates/logic/core/CoursesLogic.java [hunk 4]
```diff
@@ -656,12 +660,14 @@ public String getCourseStudentListAsCsv(String courseId, String googleId) throws
...
 
         StringBuilder export = new StringBuilder(100);
-        String courseInfo = "Course ID," + SanitizationHelper.sanitizeForCsv(courseId) + Const.EOL
-                      + "Course Name," + SanitizationHelper.sanitizeForCsv(course.course.getName()) + Const.EOL
-                      + Const.EOL + Const.EOL;
+        String courseInfo = "Course ID," + SanitizationHelper.sanitizeForCsv(courseId) + System.lineSeparator()
+                      + "Course Name," + SanitizationHelper.sanitizeForCsv(course.course.getName())
+                      + System.lineSeparator()
+                      + System.lineSeparator() + System.lineSeparator();
         export.append(courseInfo);
```
Comment (tshradheya) [inline src/main/java/teammates/logic/core/CoursesLogic.java]: Combine line 665 and 666

File: src/test/java/teammates/test/cases/testdriver/HtmlHelperTest.java [hunk 5]
```diff
@@ -27,17 +25,19 @@ public void testComparison() throws IOException {
...
+        actual = expected.replace("<HEAD>", "    <HEAD>    \t" + System.lineSeparator());
         HtmlHelper.assertSameHtml(expected, actual, false);
 
         //change attribute order
         actual = expected.replace("language=\"JavaScript\" src=\"a.js\"", "  src=\"a.js\"   language=\"JavaScript\"  ");
         HtmlHelper.assertSameHtml(expected, actual, false);
 
-        actual = expected.replace("<P>", "<P>\n\n" + EOL + EOL);
+        actual = expected.replace("<P>", "<P>\n\n" + System.lineSeparator()
+            + System.lineSeparator());
```
Comment (tshradheya) [inline src/test/java/teammates/test/cases/testdriver/HtmlHelperTest.java]: Combine lines 35 and 36

File: src/main/java/teammates/logic/core/CoursesLogic.java (hunk 3 above)
Comment (shivanshsoni) [reply src/main/java/teammates/logic/core/CoursesLogic.java]: Yes, they can be combined

File: src/main/java/teammates/logic/core/CoursesLogic.java (hunk 4 above)
Comment (shivanshsoni) [reply src/main/java/teammates/logic/core/CoursesLogic.java]: Ok sir

**Comment by shivanshsoni** [comment]
@tshradheya please review

File: src/test/java/teammates/test/cases/logic/StudentsLogicTest.java [hunk 6]
```diff
@@ -451,7 +451,8 @@ private void testEnrollLinesChecking() throws Exception {
 
         ______TS("enrollLines with invalid parameters");
         String invalidTeamName = StringHelperExtension.generateStringOfLength(FieldValidator.TEAM_NAME_MAX_LENGTH + 1);
-        String invalidStudentName = StringHelperExtension.generateStringOfLength(FieldValidator.PERSON_NAME_MAX_LENGTH + 1);
+        String invalidStudentName = StringHelperExtension.generateStringOfLength(FieldValidator.PERSON_NAME_MAX_LENGTH
+                                    + 1);
```
Comment (tshradheya) [inline src/test/java/teammates/test/cases/logic/StudentsLogicTest.java]: Avoid unnecessary changes

File: src/test/java/teammates/test/cases/datatransfer/InstructorAttributesTest.java [hunk 7]
```diff
@@ -195,14 +195,16 @@ public void testGetInvalidityInfo() throws Exception {
...
                 getPopulatedEmptyStringErrorMessage(
                     FieldValidator.COURSE_ID_ERROR_MESSAGE_EMPTY_STRING,
-                    FieldValidator.COURSE_ID_FIELD_NAME, FieldValidator.COURSE_ID_MAX_LENGTH) + EOL
+                    FieldValidator.COURSE_ID_FIELD_NAME, FieldValidator.COURSE_ID_MAX_LENGTH)
+                + System.lineSeparator()
                 + getPopulatedEmptyStringErrorMessage(
                       FieldValidator.SIZE_CAPPED_NON_EMPTY_STRING_ERROR_MESSAGE_EMPTY_STRING,
-                      FieldValidator.PERSON_NAME_FIELD_NAME, FieldValidator.PERSON_NAME_MAX_LENGTH) + EOL
+                      FieldValidator.PERSON_NAME_FIELD_NAME, FieldValidator.PERSON_NAME_MAX_LENGTH)
+                + System.lineSeparator()
```
Comment (tshradheya) [inline src/test/java/teammates/test/cases/datatransfer/InstructorAttributesTest.java]: Combine line 202 and 203

File: src/test/java/teammates/test/cases/datatransfer/InstructorAttributesTest.java [hunk 8]
```diff
@@ -195,14 +195,16 @@ public void testGetInvalidityInfo() throws Exception {
         errorMessage =
                 getPopulatedEmptyStringErrorMessage(
                     FieldValidator.COURSE_ID_ERROR_MESSAGE_EMPTY_STRING,
-                    FieldValidator.COURSE_ID_FIELD_NAME, FieldValidator.COURSE_ID_MAX_LENGTH) + EOL
+                    FieldValidator.COURSE_ID_FIELD_NAME, FieldValidator.COURSE_ID_MAX_LENGTH)
+                + System.lineSeparator()
```
Comment (tshradheya) [inline src/test/java/teammates/test/cases/datatransfer/InstructorAttributesTest.java]: Combine line 198 and 199

File: src/test/java/teammates/test/cases/datatransfer/InstructorAttributesTest.java [hunk 9]
```diff
@@ -175,17 +173,19 @@ public void testGetInvalidityInfo() throws Exception {
...
                 + getPopulatedEmptyStringErrorMessage(
                       FieldValidator.COURSE_ID_ERROR_MESSAGE_EMPTY_STRING,
-                      FieldValidator.COURSE_ID_FIELD_NAME, FieldValidator.COURSE_ID_MAX_LENGTH) + EOL
+                      FieldValidator.COURSE_ID_FIELD_NAME, FieldValidator.COURSE_ID_MAX_LENGTH)
+                + System.lineSeparator()
                 + getPopulatedEmptyStringErrorMessage(
                       FieldValidator.SIZE_CAPPED_NON_EMPTY_STRING_ERROR_MESSAGE_EMPTY_STRING,
-                      FieldValidator.PERSON_NAME_FIELD_NAME, FieldValidator.PERSON_NAME_MAX_LENGTH) + EOL
+                      FieldValidator.PERSON_NAME_FIELD_NAME, FieldValidator.PERSON_NAME_MAX_LENGTH)
+                + System.lineSeparator()
```
Comment (tshradheya) [inline src/test/java/teammates/test/cases/datatransfer/InstructorAttributesTest.java]: Combine line 183 and 184

File: src/test/java/teammates/test/cases/datatransfer/InstructorAttributesTest.java [hunk 10]
```diff
@@ -175,17 +173,19 @@ public void testGetInvalidityInfo() throws Exception {
                 getPopulatedErrorMessage(
                     FieldValidator.GOOGLE_ID_ERROR_MESSAGE, i.googleId,
                     FieldValidator.GOOGLE_ID_FIELD_NAME, FieldValidator.REASON_INCORRECT_FORMAT,
-                    FieldValidator.GOOGLE_ID_MAX_LENGTH) + EOL
+                    FieldValidator.GOOGLE_ID_MAX_LENGTH) + System.lineSeparator()
                 + getPopulatedEmptyStringErrorMessage(
                       FieldValidator.COURSE_ID_ERROR_MESSAGE_EMPTY_STRING,
-                      FieldValidator.COURSE_ID_FIELD_NAME, FieldValidator.COURSE_ID_MAX_LENGTH) + EOL
+                      FieldValidator.COURSE_ID_FIELD_NAME, FieldValidator.COURSE_ID_MAX_LENGTH)
+                + System.lineSeparator()
```
Comment (tshradheya) [inline src/test/java/teammates/test/cases/datatransfer/InstructorAttributesTest.java]: Combine line 179 and 180

File: src/test/java/teammates/test/cases/datatransfer/FeedbackQuestionAttributesTest.java [hunk 11]
```diff
@@ -147,16 +151,18 @@ public void testValidate() throws Exception {
...
         errorMessage = String.format(FieldValidator.PARTICIPANT_TYPE_TEAM_ERROR_MESSAGE,
                                      fq.recipientType.toDisplayRecipientName(),
-                                     fq.giverType.toDisplayGiverName()) + EOL
+                                     fq.giverType.toDisplayGiverName()) + System.lineSeparator()
                        + String.format(FieldValidator.PARTICIPANT_TYPE_ERROR_MESSAGE, fq.showGiverNameTo.get(0).toString(),
-                                       FieldValidator.VIEWER_TYPE_NAME) + EOL
-                       + "Trying to show giver name to STUDENTS without showing response first." + EOL
+                                       FieldValidator.VIEWER_TYPE_NAME) + System.lineSeparator()
+                       + "Trying to show giver name to STUDENTS without showing response first."
+                       + System.lineSeparator()
```
Comment (tshradheya) [inline src/test/java/teammates/test/cases/datatransfer/FeedbackQuestionAttributesTest.java]: No need to writing it on a separate line

File: src/test/java/teammates/test/cases/datatransfer/FeedbackQuestionAttributesTest.java [hunk 12]
```diff
@@ -147,16 +151,18 @@ public void testValidate() throws Exception {
...
                        + String.format(FieldValidator.PARTICIPANT_TYPE_ERROR_MESSAGE,
                                        fq.showRecipientNameTo.get(0).toString(),
-                                       FieldValidator.VIEWER_TYPE_NAME) + EOL
-                       + "Trying to show recipient name to STUDENTS without showing response first." + EOL
+                                       FieldValidator.VIEWER_TYPE_NAME) + System.lineSeparator()
+                       + "Trying to show recipient name to STUDENTS without showing response first."
+                       + System.lineSeparator()
```
Comment (tshradheya) [inline src/test/java/teammates/test/cases/datatransfer/FeedbackQuestionAttributesTest.java]: No need to writing it on a separate line

File: src/test/java/teammates/test/cases/datatransfer/FeedbackQuestionAttributesTest.java [hunk 13]
```diff
@@ -126,16 +128,18 @@ public void testValidate() throws Exception {
...
```
Comment (tshradheya) [inline src/test/java/teammates/test/cases/datatransfer/FeedbackQuestionAttributesTest.java]: No need to writing it on a separate line

File: src/test/java/teammates/test/cases/datatransfer/FeedbackQuestionAttributesTest.java [hunk 14]
```diff
@@ -126,16 +128,18 @@ public void testValidate() throws Exception {
...
         errorMessage = String.format(FieldValidator.PARTICIPANT_TYPE_TEAM_ERROR_MESSAGE,
                                      fq.recipientType.toDisplayRecipientName(),
-                                     fq.giverType.toDisplayGiverName()) + EOL
+                                     fq.giverType.toDisplayGiverName()) + System.lineSeparator()
                        + String.format(FieldValidator.PARTICIPANT_TYPE_ERROR_MESSAGE, fq.showGiverNameTo.get(0).toString(),
-                                       FieldValidator.VIEWER_TYPE_NAME) + EOL
-                       + "Trying to show giver name to STUDENTS without showing response first." + EOL
+                                       FieldValidator.VIEWER_TYPE_NAME) + System.lineSeparator()
+                       + "Trying to show giver name to STUDENTS without showing response first."
+                       + System.lineSeparator()
```
Comment (tshradheya) [inline src/test/java/teammates/test/cases/datatransfer/FeedbackQuestionAttributesTest.java]: No need to writing it on a separate line

File: src/test/java/teammates/test/cases/datatransfer/AccountAttributesTest.java [hunk 15]
```diff
@@ -42,15 +40,16 @@ public void testGetInvalidStateInfo() throws Exception {
         String expectedError =
                 getPopulatedEmptyStringErrorMessage(
                     FieldValidator.SIZE_CAPPED_NON_EMPTY_STRING_ERROR_MESSAGE_EMPTY_STRING,
-                    FieldValidator.PERSON_NAME_FIELD_NAME, FieldValidator.PERSON_NAME_MAX_LENGTH) + EOL
+                    FieldValidator.PERSON_NAME_FIELD_NAME, FieldValidator.PERSON_NAME_MAX_LENGTH)
+                + System.lineSeparator()
```
Comment (tshradheya) [inline src/test/java/teammates/test/cases/datatransfer/AccountAttributesTest.java]: No need to writing it on a separate line

File: src/test/java/teammates/test/cases/datatransfer/CourseAttributesTest.java [hunk 16]
```diff
@@ -106,10 +104,11 @@ public void testValidate() throws Exception {
                 getPopulatedErrorMessage(
                     FieldValidator.COURSE_ID_ERROR_MESSAGE, invalidCourse.getId(),
                     FieldValidator.COURSE_ID_FIELD_NAME, FieldValidator.REASON_TOO_LONG,
-                    FieldValidator.COURSE_ID_MAX_LENGTH) + EOL
+                    FieldValidator.COURSE_ID_MAX_LENGTH) + System.lineSeparator()
                 + getPopulatedEmptyStringErrorMessage(
                       FieldValidator.SIZE_CAPPED_NON_EMPTY_STRING_ERROR_MESSAGE_EMPTY_STRING,
-                      FieldValidator.COURSE_NAME_FIELD_NAME, FieldValidator.COURSE_NAME_MAX_LENGTH) + EOL
+                      FieldValidator.COURSE_NAME_FIELD_NAME, FieldValidator.COURSE_NAME_MAX_LENGTH)
+                + System.lineSeparator()
```
Comment (tshradheya) [inline src/test/java/teammates/test/cases/datatransfer/CourseAttributesTest.java]: No need to writing it on a separate line

File: src/test/java/teammates/test/cases/datatransfer/FeedbackQuestionAttributesTest.java (hunk 13 above)
Comment (shivanshsoni) [reply src/test/java/teammates/test/cases/datatransfer/FeedbackQuestionAttributesTest.java]: Sir, if I am writing it in the same line then it increases line limit of 125 columns

File: src/test/java/teammates/test/cases/datatransfer/FeedbackQuestionAttributesTest.java (hunk 12 above)
Comment (shivanshsoni) [reply src/test/java/teammates/test/cases/datatransfer/FeedbackQuestionAttributesTest.java]: Sir, if I am writing it in the same line then it increases line limit of 125 columns

File: src/test/java/teammates/test/cases/datatransfer/FeedbackQuestionAttributesTest.java (hunk 13 above)
Comment (tshradheya) [reply src/test/java/teammates/test/cases/datatransfer/FeedbackQuestionAttributesTest.java]: Sorry my bad.

Comment (shivanshsoni) [reply src/test/java/teammates/test/cases/datatransfer/FeedbackQuestionAttributesTest.java]: No problem Sir, waiting to pass travis then review my chnages please.

Comment (tshradheya) [reply src/test/java/teammates/test/cases/datatransfer/FeedbackQuestionAttributesTest.java]: Also, update your branch with the latest master branch

**Comment by shivanshsoni** [comment]
@tshradheya Please review my changes..

File: src/test/java/teammates/test/cases/logic/StudentsLogicTest.java [hunk 17]
```diff
@@ -463,9 +463,11 @@ private void testEnrollLinesChecking() throws Exception {
         String lineWithInvalidTeamNameAndStudentNameAndEmail =
                 invalidTeamName + "|" + invalidStudentName + "|" + "invalid_email.tmt";
 
-        enrollLines = headerLine + Const.EOL + lineWithInvalidTeamName + Const.EOL + lineWithInvalidStudentName
-                    + Const.EOL + lineWithInvalidEmail + Const.EOL + lineWithInvalidStudentNameAndEmail + Const.EOL
-                    + lineWithInvalidTeamNameAndEmail + Const.EOL + lineWithInvalidTeamNameAndStudentNameAndEmail;
+        enrollLines = headerLine + System.lineSeparator() + lineWithInvalidTeamName + System.lineSeparator()
```
Comment (LiHaoTan) [inline src/test/java/teammates/test/cases/logic/StudentsLogicTest.java]: Not your fault, but this is a little hard. Let's break after every `System.lineSeparator()`.

File: src/test/java/teammates/test/cases/logic/StudentsLogicTest.java [hunk 18]
```diff
@@ -522,7 +524,8 @@ private void testEnrollLinesChecking() throws Exception {
         String lineWithExtraParameters = "Team 4 | StudentWithExtraParameters | "
                 + " studentWithExtraParameters@email.tmt | comment | extra_parameter";
 
-        enrollLines = headerLine + Const.EOL + lineWithNoEmailInput + Const.EOL + lineWithExtraParameters;
+        enrollLines = headerLine + System.lineSeparator() + lineWithNoEmailInput + System.lineSeparator()
```
Comment (LiHaoTan) [inline src/test/java/teammates/test/cases/logic/StudentsLogicTest.java]: For consistency let's break after every `System.lineSeparator()` as well.

File: src/test/java/teammates/test/cases/logic/StudentsLogicTest.java [hunk 19]
```diff
@@ -569,7 +572,8 @@ private void testEnrollLinesChecking() throws Exception {
         String lineWithCorrectInput = "Team 3 | Mary | mary@email.tmt";
         String lineWithCorrectInputWithComment = "Team 4 | Benjamin | benjamin@email.tmt | Foreign student";
 
-        enrollLines = headerLine + Const.EOL + lineWithCorrectInput + Const.EOL + lineWithCorrectInputWithComment;
+        enrollLines = headerLine + System.lineSeparator() + lineWithCorrectInput + System.lineSeparator()
```
Comment (LiHaoTan) [inline src/test/java/teammates/test/cases/logic/StudentsLogicTest.java]: Break after `System.lineSeparator()`.

File: src/test/java/teammates/test/cases/logic/StudentsLogicTest.java [hunk 20]
```diff
@@ -578,18 +582,19 @@ private void testEnrollLinesChecking() throws Exception {
 
         ______TS("enrollLines with duplicate emails");
 
-        enrollLines = headerLine + Const.EOL + lineWithCorrectInput + Const.EOL + lineWithCorrectInput;
+        enrollLines = headerLine + System.lineSeparator() + lineWithCorrectInput + System.lineSeparator()
```
Comment (LiHaoTan) [inline src/test/java/teammates/test/cases/logic/StudentsLogicTest.java]: Break after `System.lineSeparator()`.

File: src/test/java/teammates/test/cases/logic/StudentsLogicTest.java [hunk 21]
```diff
@@ -578,18 +582,19 @@ private void testEnrollLinesChecking() throws Exception {
...
+                                 + lineWithCorrectInput;
 
         invalidInfoString = getExceptionMessageOnCreatingStudentsList(enrollLines, courseId);
 
         expectedInvalidInfoString = "Same email address as the student in line \"" + lineWithCorrectInput + "\"";
         AssertHelper.assertContains(expectedInvalidInfoString, invalidInfoString);
 
         ______TS("enrollLines with a mix of all above cases");
-        enrollLines = headerLine + Const.EOL + lineWithInvalidTeamName + Const.EOL
+        enrollLines = headerLine + System.lineSeparator() + lineWithInvalidTeamName + System.lineSeparator()
```
Comment (LiHaoTan) [inline src/test/java/teammates/test/cases/logic/StudentsLogicTest.java]: Break after `System.lineSeparator()`.

File: src/test/java/teammates/test/cases/logic/StudentsLogicTest.java [hunk 22]
```diff
@@ -686,9 +691,10 @@ private void testEnrollStudents() throws Exception {
         String line2 = "\u00A0t3  |n3|  e3@g|c3  ";
         String line3 = "t4|n4|  e4@g|c4";
         String line4 = "t5|  n5|e5@g  |c5";
-        String lines = headerLine + Const.EOL + line0 + Const.EOL + line1 + Const.EOL + line2 + Const.EOL
-                    + "  \t \t \t \t           " + Const.EOL + line3 + Const.EOL + Const.EOL + line4
-                    + Const.EOL + "    " + Const.EOL + Const.EOL;
+        String lines = headerLine + System.lineSeparator() + line0 + System.lineSeparator() + line1
```
Comment (LiHaoTan) [inline src/test/java/teammates/test/cases/logic/StudentsLogicTest.java]: Break after `System.lineSeparator()`. Consecutive ones need not break.

File: src/test/java/teammates/test/cases/logic/StudentsLogicTest.java [hunk 23]
```diff
@@ -710,7 +716,8 @@ private void testEnrollStudents() throws Exception {
 
         String modifiedLine2 = "t3|modified name|e3@g|c3";
         String line5 = "t6|n6|e6@g|c6";
-        lines = headerLine + Const.EOL + line0 + Const.EOL + modifiedLine2 + Const.EOL + line1 + Const.EOL + line5;
+        lines = headerLine + System.lineSeparator() + line0 + System.lineSeparator() + modifiedLine2
```
Comment (LiHaoTan) [inline src/test/java/teammates/test/cases/logic/StudentsLogicTest.java]: Break after `System.lineSeparator()`.

File: src/test/java/teammates/test/cases/logic/StudentsLogicTest.java [hunk 24]
```diff
@@ -731,8 +738,8 @@ private void testEnrollStudents() throws Exception {
 
         // no changes should be done to the database
         String incorrectLine = "incorrectly formatted line";
-        lines = headerLine + Const.EOL + "t7|n7|e7@g|c7" + Const.EOL + incorrectLine + Const.EOL + line2 + Const.EOL
-                + line3;
+        lines = headerLine + System.lineSeparator() + "t7|n7|e7@g|c7" + System.lineSeparator()
```
Comment (LiHaoTan) [inline src/test/java/teammates/test/cases/logic/StudentsLogicTest.java]: Break after `System.lineSeparator()`.

File: src/test/java/teammates/test/cases/testdriver/HtmlHelperTest.java [hunk 25]
```diff
@@ -27,17 +25,18 @@ public void testComparison() throws IOException {
...
         //change attribute order
         actual = expected.replace("language=\"JavaScript\" src=\"a.js\"", "  src=\"a.js\"   language=\"JavaScript\"  ");
         HtmlHelper.assertSameHtml(expected, actual, false);
 
-        actual = expected.replace("<P>", "<P>\n\n" + EOL + EOL);
+        actual = expected.replace("<P>", "<P>\n\n" + System.lineSeparator() + System.lineSeparator());
         HtmlHelper.assertSameHtml(expected, actual, false);
 
-        actual = expected.replace("<DIV></DIV></DIV>", EOL + EOL + "\n<DIV>\n\n</DIV></DIV>\n\n" + EOL);
+        actual = expected.replace("<DIV></DIV></DIV>", System.lineSeparator()
```
Comment (LiHaoTan) [inline src/test/java/teammates/test/cases/testdriver/HtmlHelperTest.java]: Break before the first `System.lineSeparator()`.

Comment (shivanshsoni) [reply src/test/java/teammates/test/cases/testdriver/HtmlHelperTest.java]: Actually I tried this but line size is increasing and going more than 125 chars per line

Comment (LiHaoTan) [reply src/test/java/teammates/test/cases/testdriver/HtmlHelperTest.java]: That's not true?

Comment (shivanshsoni) [reply src/test/java/teammates/test/cases/testdriver/HtmlHelperTest.java]: Sorry, my bad

**Comment by shivanshsoni** [comment]
@LiHaoTan Changes Done, Please check..

**Comment by LiHaoTan** [comment]
@shivanshsoni Please don't rebase next time when the changes are so big, it makes reviewing difficult.

**Comment by shivanshsoni** [comment]
I will take care of this next time..

File: src/test/java/teammates/test/cases/logic/StudentsLogicTest.java [hunk 26]
```diff
@@ -522,7 +526,9 @@ private void testEnrollLinesChecking() throws Exception {
         String lineWithExtraParameters = "Team 4 | StudentWithExtraParameters | "
                 + " studentWithExtraParameters@email.tmt | comment | extra_parameter";
 
-        enrollLines = headerLine + Const.EOL + lineWithNoEmailInput + Const.EOL + lineWithExtraParameters;
+        enrollLines = headerLine + System.lineSeparator()
+                                 + lineWithNoEmailInput + System.lineSeparator()
```
Comment (LiHaoTan) [inline src/test/java/teammates/test/cases/logic/StudentsLogicTest.java]: Use 8 spaces indentation, not custom.

File: src/test/java/teammates/test/cases/logic/StudentsLogicTest.java [hunk 27]
```diff
@@ -569,7 +575,9 @@ private void testEnrollLinesChecking() throws Exception {
         String lineWithCorrectInput = "Team 3 | Mary | mary@email.tmt";
         String lineWithCorrectInputWithComment = "Team 4 | Benjamin | benjamin@email.tmt | Foreign student";
 
-        enrollLines = headerLine + Const.EOL + lineWithCorrectInput + Const.EOL + lineWithCorrectInputWithComment;
+        enrollLines = headerLine + System.lineSeparator()
+                                 + lineWithCorrectInput + System.lineSeparator()
```
Comment (LiHaoTan) [inline src/test/java/teammates/test/cases/logic/StudentsLogicTest.java]: 8 spaces indentation

File: src/test/java/teammates/test/cases/logic/StudentsLogicTest.java [hunk 28]
```diff
@@ -578,18 +586,22 @@ private void testEnrollLinesChecking() throws Exception {
 
         ______TS("enrollLines with duplicate emails");
 
-        enrollLines = headerLine + Const.EOL + lineWithCorrectInput + Const.EOL + lineWithCorrectInput;
+        enrollLines = headerLine + System.lineSeparator()
+                                 + lineWithCorrectInput + System.lineSeparator()
```
Comment (LiHaoTan) [inline src/test/java/teammates/test/cases/logic/StudentsLogicTest.java]: 8 spaces indentation.

File: src/test/java/teammates/test/cases/logic/StudentsLogicTest.java [hunk 29]
```diff
@@ -686,9 +698,14 @@ private void testEnrollStudents() throws Exception {
         String line2 = "\u00A0t3  |n3|  e3@g|c3  ";
         String line3 = "t4|n4|  e4@g|c4";
         String line4 = "t5|  n5|e5@g  |c5";
-        String lines = headerLine + Const.EOL + line0 + Const.EOL + line1 + Const.EOL + line2 + Const.EOL
-                    + "  \t \t \t \t           " + Const.EOL + line3 + Const.EOL + Const.EOL + line4
-                    + Const.EOL + "    " + Const.EOL + Const.EOL;
+        String lines = headerLine + System.lineSeparator()
+                    + line0 + System.lineSeparator()
```
Comment (LiHaoTan) [inline src/test/java/teammates/test/cases/logic/StudentsLogicTest.java]: 8 spaces indentation

File: src/test/java/teammates/test/cases/logic/StudentsLogicTest.java [hunk 30]
```diff
@@ -710,7 +727,11 @@ private void testEnrollStudents() throws Exception {
 
         String modifiedLine2 = "t3|modified name|e3@g|c3";
         String line5 = "t6|n6|e6@g|c6";
-        lines = headerLine + Const.EOL + line0 + Const.EOL + modifiedLine2 + Const.EOL + line1 + Const.EOL + line5;
+        lines = headerLine + System.lineSeparator()
+                           + line0 + System.lineSeparator()
```
Comment (LiHaoTan) [inline src/test/java/teammates/test/cases/logic/StudentsLogicTest.java]: 8 spaces indentation.

**Comment by shivanshsoni** [comment]
@LiHaoTan I have done the required changes. Please check Sir

**Comment by LiHaoTan** [comment]
@shivanshsoni Why are you rebasing again?

**Comment by shivanshsoni** [comment]
Sorry Sir, I will definitely take care of it next time

**Comment by LiHaoTan** [comment]
No worries, I can still do a diff, though it requires a bit more work on my part :).

File: src/test/java/teammates/test/cases/logic/StudentsLogicTest.java [hunk 31]
```diff
@@ -463,9 +463,13 @@ private void testEnrollLinesChecking() throws Exception {
         String lineWithInvalidTeamNameAndStudentNameAndEmail =
                 invalidTeamName + "|" + invalidStudentName + "|" + "invalid_email.tmt";
 
-        enrollLines = headerLine + Const.EOL + lineWithInvalidTeamName + Const.EOL + lineWithInvalidStudentName
-                    + Const.EOL + lineWithInvalidEmail + Const.EOL + lineWithInvalidStudentNameAndEmail + Const.EOL
-                    + lineWithInvalidTeamNameAndEmail + Const.EOL + lineWithInvalidTeamNameAndStudentNameAndEmail;
+        enrollLines = headerLine + System.lineSeparator()
+                    + lineWithInvalidTeamName + System.lineSeparator()
```
Comment (LiHaoTan) [inline src/test/java/teammates/test/cases/logic/StudentsLogicTest.java]: 8 spaces?

File: src/test/java/teammates/test/cases/logic/StudentsLogicTest.java [hunk 32]
```diff
@@ -539,9 +545,9 @@ private void testEnrollLinesChecking() throws Exception {
         String lineWithStudentNameEmpty = "Team 5 |  | no_name@email.tmt";
         String lineWithEmailEmpty = "Team 5 | StudentWithEmailFieldEmpty | |";
 
-        enrollLines = headerLine + Const.EOL
-                      + lineWithTeamNameEmpty + Const.EOL
-                      + lineWithStudentNameEmpty + Const.EOL
+        enrollLines = headerLine + System.lineSeparator()
+                      + lineWithTeamNameEmpty + System.lineSeparator()
```
Comment (LiHaoTan) [inline src/test/java/teammates/test/cases/logic/StudentsLogicTest.java]: 8 spaces?

**Comment by shivanshsoni** [comment]
@LiHaoTan Please review Sir

File: src/main/java/teammates/common/datatransfer/questions/FeedbackRankOptionsQuestionDetails.java [hunk 33]
```diff
@@ -358,7 +358,7 @@ public String getQuestionResultStatisticsCsv(
         });
 
         return "Option, Overall Rank, Ranks Received" + System.lineSeparator()
-                + fragments.toString() + System.lineSeparator();
+            + fragments.toString() + System.lineSeparator();
```
Comment (LiHaoTan) [inline src/main/java/teammates/common/datatransfer/questions/FeedbackRankOptionsQuestionDetails.java]: Extraneous changes?

**Comment by shivanshsoni** [comment]
why travis is failing May I know??

**Comment by shivanshsoni** [comment]
@LiHaoTan please check

**Comment by LiHaoTan** [comment]
@shivanshsoni that is likely a network error (I restarted the build to be sure that it isn't failing for other reasons).

**Comment by shivanshsoni** [comment]
I am very happy after contrbuting, Can solve other issues now. :-)
